deploy/common/k8sbased.sh

```diff
@@ -130,6 +130,15 @@ function launch_central {
 
     echo "Generating central config..."
 
+
+    if [[ -z "$CI" && -z "${CENTRAL_HELM_DEPLOY:-}" && -x "$(command -v helm)" && "$(helm version --short)" == v3.* ]]; then
```

Comment:
the check for CENTRAL_HELM_DEPLOY looks suspicious. More specifically, the default empty value seems to somehow double up with the -z check. The accompaniing text also implies that I could set the value to true, but that is not true either. 

```
CENTRAL_HELM_DEPLOY=true; test -z "${CENTRAL_HELM_DEPLOY:-}"; echo $? 
```

Maybe I am too picky here?

Reply:
Good catch. The same code is used for Sensor below in L564, I would prefer here to be pragmatic in this scenario as it is rare that locally Helm is disabled. 

Here:
```
    if [[ -z "$CI" && -z "${SENSOR_HELM_DEPLOY:-}" && -x "$(command -v helm)" && "$(helm version --short)" == v3.* ]]; then
      echo >&2 "================================================================================================"
      echo >&2 "NOTE: Based on your environment, you are using the Helm-based deployment method."
      echo >&2 "      To disable the Helm based installation set SENSOR_HELM_DEPLOY=false"
      echo >&2 "================================================================================================"
      SENSOR_HELM_DEPLOY=true
    fi
```